appveyor.yml

```diff
@@ -1,5 +1,15 @@
 version: 1.0.{build}
 
+install:
+  - git submodule update --init --recursive  
+
 build_script:
-  - git submodule update --init --recursive
   - vcbuild.bat MSVC2013
+  
+before_test:
+  - gem install cucumber
```

Comment:
We don't need these since the integration tests has been removed from shared/mson branch and move to apiaryio/drafter